cli/tests/unit/serve_test.ts

```diff
@@ -985,7 +982,7 @@ function createStreamTest(count: number, delay: number, action: string) {
       }
     } finally {
       ac.abort();
-      await server.finished;
+      await server.shutdown();
```

Comment:
Slowly migrating tests  to the new graceful shutdown API to avoid possible flakes.